### PR TITLE
docs: How to use devcontainers and a local Kubernetes cluster

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge_turrado@hotmail.es>

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -219,6 +219,10 @@ If you prefer to use an authenticated user, you can use a user or service accoun
 
 ## Miscellaneous
 
+### How to use devcontainers and a local Kubernetes cluster
+
+If you are working with devcontainers, Visual Studio Code is running from the container, so if you are using any local Kubernetes cluster like Kind or minikube, you don't have access from the container as default. To solve this and be able to work with devcontainers and a local cluster, you should follow [this official documentation from Microsoft](https://github.com/Microsoft/vscode-dev-containers/tree/main/containers/kubernetes-helm).
```
Comment (tomkerkhove) [inline BUILD.md:224]: The first phrase is a bit of an odd one and I'm not sure what you mean.

Is this what you mean?
```suggestion
If you are using Visual Studio Code to [develop inside a container (by using `devcontainers.json`)](https://code.visualstudio.com/docs/remote/containers) you can use any local Kubernetes cluster such as Kind or minikube. However, the development container will not have access by default.

To solve this and be able to work with `devcontainers.json`and a local cluster, you should follow [this official documentation from Microsoft](https://github.com/Microsoft/vscode-dev-containers/tree/main/containers/kubernetes-helm).
```

Comment (JorTurFer) [reply BUILD.md]: I have rephrased the paragraph to be clearer, PTAL

**Review by tomkerkhove — APPROVED** [review]
You're assuming people know what devcontainers are but that's fine for me

**Comment by JorTurFer** [comment]
> You're assuming people know what devcontainers are but that's fine for me

You are right xD
I have added a link to devcontainers info to clarify it just in case

**Pull request merged**